stories/feedback/Banner.stories.js

```diff
@@ -1,66 +1,95 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
 import { action } from '@storybook/addon-actions';
-import StoryContainer from '../storyHelpers/StoryContainer';
 
-import { Banner } from '@sparkpost/matchbox';
+import { ThemeProvider, Banner } from '@sparkpost/matchbox';
 
-const primaryAction = {
-  content: 'Learn More',
-  onClick: action('Learn More Clicked')
+addDecorator(storyFn => <ThemeProvider>{storyFn()}</ThemeProvider>);
+
+export default {
+  title: 'Feedback|Banner',
 };
 
-storiesOf('Feedback|Banner', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{ getStory() }</StoryContainer>
-  ))
+export const DefaultBanner = withInfo()(() => (
+  <Banner title="Heads up!" onDismiss={action('Dismiss Clicked')}>
+    If you're importing recipients from a previous provider, be sure to also{' '}
+    <a href="https://design.sparkpost.com">import your suppressions</a> later.
+  </Banner>
+));
 
-  .add('Default', withInfo()(() => (
-    <Banner title='Heads up!' >
-      If you're importing recipients from a previous provider, be sure to also <a>import your suppressions</a> later.
-    </Banner>
-  )))
+export const SuccessBanner = withInfo()(() => (
+  <Banner
+    title="Thank you for your dedicated IP purchase"
+    status="success"
+    onDismiss={action('Success Dismiss Clicked')}
+  >
+    Happy Sending!
+  </Banner>
+));
 
-  .add('Success', withInfo()(() => (
-    <Banner
-      title='Thank you for your dedicated IP purchase'
-      status='success'
-      onDismiss={action('Success Dismiss Clicked')} >
-      Happy Sending!
-    </Banner>
-  )))
+export const InfoBanner = withInfo()(() => (
+  <Banner
+    status="info"
+    title="Verify your email address"
+    action={{
+      content: 'Resend Email',
+      onClick: action('Resend Email Clicked'),
+    }}
+  >
+    Please click the link in the email we sent you to unlock the higher daily sending limits.
+  </Banner>
+));
```

Comment:
In these examples, I think ideally the child element would be a `<p>` to be semantically correct - that will encourage future devs to do the same instead of just a string.